tests/components/awair/const.py

```diff
@@ -0,0 +1,20 @@
+"""Constants used in Awair tests."""
+
+import json
+
+from homeassistant.const import CONF_ACCESS_TOKEN
+
+from tests.common import load_fixture
+
+AWAIR_UUID = "awair_24947"
+CONFIG = {CONF_ACCESS_TOKEN: "12345"}
+CONF_UNIQUE_ID = "foo@bar.com"
```

Comment:
We only use `CONF_` constants for config keys, not for values.

Reply:
Sure.

34340847cc